net-im/spectrum2/spectrum2-2.0.6.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils user
+
+DESCRIPTION="An open source instant messaging transport"
+HOMEPAGE="https://www.spectrum.im"
+SRC_URI="https://github.com/SpectrumIM/spectrum2/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug doc frotz irc mysql postgres purple sms sqlite test twitter xmpp"
+REQUIRED_USE="|| ( mysql postgres sqlite )"
+
+DEPEND="dev-libs/boost
+	dev-libs/expat
+	dev-libs/libxml2
+	dev-libs/log4cxx
+	dev-libs/jsoncpp
+	dev-libs/openssl:0=
+	dev-libs/popt
+	dev-libs/protobuf
+	net-dns/libidn
+	sys-libs/zlib
+	doc? ( app-doc/doxygen )
+	frotz? ( !games-engines/frotz )
+	irc? ( net-im/libcommuni )
+	mysql? ( || ( dev-db/mariadb-connector-c dev-db/mysql-connector-c ) )
+	postgres? ( dev-db/postgresql:= )
+	purple? ( dev-libs/glib
+		dev-libs/libev
+		net-im/pidgin )
+	sms? ( app-mobilephone/smstools )
+	sqlite? ( dev-db/sqlite )
+	test? ( dev-util/cppunit )
+	xmpp? ( net-im/swiften )"
+
+RDEPEND="${DEPEND}"
+
+pkg_setup() {
+	enewgroup spectrum
+	enewuser spectrum -1 -1 /var/lib/spectrum2 spectrum
+}
+
+src_prepare() {
+	eapply "${FILESDIR}"/${P}-add_qt5_libcommuni.patch
+
+	cmake-utils_src_prepare
+}
+
+src_configure() {
+	if use debug; then
+		CMAKE_BUILD_TYPE="Debug"
+	fi
+
+	local mycmakeargs=(
+		-DENABLE_DOCS="$(usex doc ON OFF)" \
```

Comment:
You don't need to force `ON OFF` for most of CMake projects. It handles the default yes/no just as well.

Reply:
Even, when CMakeLists.txt uses itself ON/OFF?

Reply:
Yes. As long as they use `if (FOO)` CMake automagically handles different values for on/off, yes/no, etc. You'd need explicit on/off only if they did direct string comparison like `IF (FOO STREQUAL "ON")`

Reply:
Changed to `YES` and `NO`. But most eBuilds seems to use `ON` and `OFF`?

Reply:
Okay, now I got it ;-)